### ISSUE TITLE
Enabling extension breaks launching extension development host 

### ISSUE THREAD

**Issue by disco0** [DeltaXML/vscode-xslt-tokenizer#67]
Extension Version: `0.2.9`

<details>
<summary>VS Code Information</summary><p>

```
Version: 1.57.0-insider
Commit: 0b355ffc89b2020a941ce149f2704acf13a49afd
Date: 2021-05-20T17:44:35.872Z (11 hrs ago)
Electron: 12.0.7
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Darwin x64 18.7.0
```

</p></details>

Opened up another extension project, launched the extension development host and nothing happens, not even the prelaunch task—based on dev tools output I tried disabling the extension and everything went back to normal, tried reinstalling and the issue still presented after re-enabling.

Related error output in dev tools:

<img width="1200" alt="deltaxml.xslt-path error log" src="https://user-images.githubusercontent.com/13393582/119089389-e8777600-b9be-11eb-8fd0-56f5509a929d.png">

**Comment by pgfearo** [comment]
Thanks for reporting this issue.

There appear to be 3 parts to this problem that I need to resolve:

1. Understand why the XSLT/XPath extension is being triggered in this context, and explore whether it's possible to stop this happening.
2. The TaskProviders attempt to parse the tasks.json file which is not standard JSON, it is JSONC. There is a very crude attempt to 'fix' the JSONC so it can be parsed with the JSON parser - but we should move to using [Microsofts JSONC parser](https://github.com/microsoft/node-jsonc-parser).
3. The exception case where the JSONC cannot still be passed should be handled properly.

**Comment by pgfearo** [comment]
Fixed in release v0.3.0
